style.css

```diff
@@ -107,3 +110,357 @@ body {
+  width: 200px;
+}
+
+.frame2 h2 {
+  padding-right: 20px;
+}
+
+ul li::before {
+  background-color: #c1c7d0;
+}
+
+.card-role li {
+  color: #7a869a;
+}
+
+.description {
+  width: 295px;
+  height: 76px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.languages {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 203px;
+  height: 24px;
+  list-style: none;
+}
+
+.languages li {
+  padding: 4px 12px;
+  width: 53px;
+  height: 24px;
+
+  /* Primary/B50 */
+
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.word-style {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+
+  /* identical to box height, or 133% */
+
+  letter-spacing: 0.03em;
+
+  /* Primary/B400 */
+
+  color: #6070ff;
+}
+
+li:last-child {
+  width: 89px;
+}
+
+.action {
+  width: 295px;
+  height: 72px;
+}
+
+.action-button {
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  left: 0;
+  top: calc(50% - 48px / 2);
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+}
+
+.action-button:hover {
+  background-color: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  color: #fff;
+}
+
+.action-button:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.action-button:disabled {
+  border: 1px solid #c1c7d0;
+  color: #fff;
+}
+
+/* about my self section */
+.about-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 24px;
+  position: absolute;
+  width: 375px;
+  height: 1125.15px;
+  margin-top: 114px;
+  background-color: #fff;
+  border-top-right-radius: 80px;
+}
+
+.about-myself {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  width: 327px;
+  height: 409px;
+}
+
+.about-heading {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.about-intro {
+  width: 327px;
+  height: 144px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-bottom: 12px;
+}
+
+.socialmedia-container {
+  margin-left: -18px;
+}
+
+.space-icon {
+  padding-top: -30px;
+}
+
+.btn-container {
+  width: 375px;
+  height: 1263px;
+  margin-bottom: 50px;
+}
+
+.resume-btn {
+  width: 135px;
+  height: 48px;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  color: #396df2;
+  margin-top: -10px;
+}
+
+.btn-text {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  text-align: center;
+  color: #396df2;
+}
+
+.skills-section {
+  display: flex;
+}
+
+.about-languages {
+  width: 327px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-top: 50px;
+  margin-bottom: 36px;
+  list-style: none;
+}
```

Comment:
- [ ] As mentioned above, please don't fix the `width`, you can use a percentage instead and control it with margins and paddings. 😁